redisgears_v8_plugin/src/v8_backend.rs

```diff
@@ -311,6 +304,16 @@ impl BackendCtxInterfaceInitialised for V8Backend {
                     log(compiled_message);
                     script_ctx.compiled_library_api.log(compiled_message);
```

Comment:
Why do we need to log here twice? We should always use the `compiled_library_api` for logs so we will get the library name of the log message. the `log` function is just for cases where we want to log general messages and or if we do not have `compiled_library_api` (like if the engine want to log some general message).